scripts/ci/ansible/roles/jenkins_master/tasks/main.yml

```diff
@@ -5,13 +5,25 @@
   with_items:
     - id_rsa
     - id_rsa.pub
+  tags:
```

Comment:
nit: how about renaming the parent folder from jenkins_master to jenkins?

Reply:
jenkins_master role is only used to deploy jenkins master, so let the directory named like this